rts/at/annotation/README.md

```diff
@@ -0,0 +1,63 @@
+OVERVIEW
+--------
+An example of how to use REST-AT Annotation Example as an individual micro service.It's main purpose is to show how to use the [annotations](https://github.com/jbosstm/narayana/tree/main/rts/at/tx/src/main/java/org/jboss/jbossts/star/annotation).
+
+USAGE
+--------------------
+
+Prior to running the example make sure that the [REST-AT coordinator is deployed](../README.md#usage).
+
+Once REST-AT coordinator is deployed build and run the annotation example application to run on port 8082,8083,8084 as it in configured in application.properties
+    
+    cd <narayana-repo>/rts/at/annotation/flight-service
+    mvn clean package -Dquarkus.package.type=uber-jar
+    mvn quarkus:dev
+
+    repeat the same step for hotel-service and trip-service
+     
+    This will start three micro service on three diffrent ports.
```

Comment:
> Please remove initial spaces here, these are not command line instructions.

@marcosgopen I've removed the said space, please check and resolve the conversation.Thanks.

Reply:
Actually I was talking about the spaces before 'repeat the same step for hotel-service and trip-service' and 'This will start three micro service on three diffrent ports.' BTW I have just noted the typo in 'different'.

Reply:
If you have a look at the readme here 'https://github.com/moulalis/quickstart-1/tree/JBTM-3462/rts/at/annotation' you will note the 2 lines are in the code block because of the space:
'         repeat the same step for hotel-service and trip-service' -> code block
'repeat the same step for hotel-service and trip-service' ->  NO code block

Reply:
> Actually I was talking about the spaces before 'repeat the same step for hotel-service and trip-service' and 'This will start three micro service on three diffrent ports.' BTW I have just noted the typo in 'different'.

@marcosgopen Actually that space is required for formatting in README.

Reply:
Markdown collapses white space so you'd need to use `&nbsp` if you don't want it to do that.
But even though it collapses white space, it's best to keep the source document readable by human eyes.

Reply:
But English grammar does require white space after full stops and commas.